### PR TITLE
New Spanish translation

### PR DESCRIPTION
**Pull request by ftortoriello**
# Description

New complete Spanish translation, updated to the current git revision.

# Checklist

I have:

- [x] followed the project's [contributing guidelines](https://github.com/dosbox-staging/dosbox-staging/blob/master/CONTRIBUTING.md) and [code of conduct](https://github.com/dosbox-staging/dosbox-staging/blob/master/CODE_OF_CONDUCT.md).
- [x] performed a self-review of my code.
- [ ] commented on the particularly hard-to-understand areas of my code.
- [ ] split my work into well-defined, bisectable commits, and I [named my commits well](https://github.com/dosbox-staging/dosbox-staging/blob/main/CONTRIBUTING.md#commit-messages).
- [ ] applied the appropriate labels (bug, enhancement, refactoring, documentation, etc.)
- [ ] [checked](https://github.com/dosbox-staging/dosbox-staging/blob/main/scripts/compile_commits.sh) that all my commits can be built.
- [ ] confirmed that my code does not cause performance regressions (e.g., by running the Quake benchmark).
- [ ] added unit tests where applicable to prove the correctness of my code and to avoid future regressions.
- [ ] made corresponding changes to the documentation or the website according to the [documentation guidelines](https://github.com/dosbox-staging/dosbox-staging/blob/main/DOCUMENTATION.md).
- [ ] [locally verified](https://github.com/dosbox-staging/dosbox-staging/blob/main/DOCUMENTATION.md#previewing-documentation-changes-locally) my website or documentation changes.

### CODE REVIEW

**Comment by ftortoriello** [comment]
I will update this soon to keep it in sync with style changes in #3318

File: contrib/resources/translations/es.lng [hunk 1]
```diff
@@ -1,1104 +1,2567 @@
...
 .
 :MT32_NO_SUPPORTED_MODELS
-No supported models present.
+No hay modelos soportados
 .
 :MT32_ROM_NOT_LOADED
-No ROM is currently loaded
+No hay una ROM cargada actualmente
 .
 :MT32_INVENTORY_TABLE_MISSING_LETTER
```
Comment (ftortoriello) [inline contrib/resources/translations/es.lng:1339]: Both of these strings are unused in the code

**Comment by johnnovak** [comment]
Are you planning to finish this @ftortoriello? RC2 has been released with your very nice overall English help message improvements 🎉

**Comment by ftortoriello** [comment]
Sorry, I was on vacation :mountain:
I've just rebased it and updated it with the latest commits, I think it's ready :smile:

**Comment by johnnovak** [comment]
Could you update the branch again please @ftortoriello ? Sorry we forgot about this, it did not go into final 0.81.0 release. If we need to release a patch, it will be included in that (or the next stable release).

**Comment by ftortoriello** [comment]
Ok, done! No problem :)

**Review by johnnovak — APPROVED** [review]
Thanks @ftortoriello , merging 😄

**Comment by ftortoriello** [comment]
You're welcome! :smile:

**Pull request merged**